platform/features/table/src/controllers/MCTTableController.js

```diff
@@ -165,17 +165,8 @@ define(
 
             //No need to scroll
             if (this.$scope.displayRows.length < this.maxDisplayRows) {
-                //Check whether need to resynchronize visible with display
-                // rows (if data added)
-                if (this.$scope.visibleRows.length !==
-                    this.$scope.displayRows.length){
```

Comment:
@akhenry - I'm curious if there are known cases where this is an important/necessary performance optimization (don't want to introduce a regression)